src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.dtos
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@Schema(description = "Adjudication to migrate into service")
+data class AdjudicationMigrateDto(
+  @Schema(description = "The agency incident id")
+  val agencyIncidentId: Long,
+  @Schema(description = "The oic incident id")
+  val oicIncidentId: Long,
+  @Schema(description = "the offence sequence")
+  val offenceSequence: Long,
+  @Schema(description = "booking id on adjudication")
+  val bookingId: Long,
+  @Schema(description = "agency id on adjudication")
+  val agencyId: String,
+  @Schema(description = "the date and time of the incident")
+  val incidentDateTime: LocalDateTime,
+  @Schema(description = "the internal location id of the incident")
+  val locationId: Long,
+  @Schema(description = "the incident statement")
+  val statement: String,
+  @Schema(description = "adjudication under yoi rules")
+  val yoiRulesUsed: Boolean,
+  @Schema(description = "prisoner details")
+  val prisoner: MigratePrisoner,
+  @Schema(description = "offence details")
+  val offence: MigrateOffence,
+  @Schema(description = "optional victim", required = false)
+  val victim: MigrateVictim? = null,
```

Comment:
NOMIS does have multiple victims - we can provide one

Reply:
thanks!  so i will raise this with John, as to what to do about it.